src/scenicplus/cli/scenicplus.py

```diff
@@ -132,6 +133,10 @@ def prepare_menr_data(arg):
         "--multiome_mudata_fname", dest="multiome_mudata_fname",
         action="store", type=pathlib.Path, required=True,
         help="Path to multiome MuData object (from scenicplus prepare_GEX_ACC).")
+    parser.add_argument(
+        "--path_to_regions_to_subset", dest="path_to_regions_to_subset",
+        action="store", type=str, required=False,default ="",
+        help="Path to bed file for regions to subset when merging cistromes (MACS called peaks).")
```

Comment:
They can be any bed file right, don't necessarily need to be peaks called by MACS? If this is the case I would remove "`(MACS called peaks)`" from the documentation to avoid confusion.

Reply:
True could be removed. or could add for example. Will fix it